packages/migration/package.json

```diff
@@ -2,28 +2,15 @@
   "author": "Fewlines",
   "description": "Migration part of Fewlines Web Libraries",
   "dependencies": {
-    "@fwl/database": "0.6.4",
-    "uuid": "8.3.2"
-  },
-  "devDependencies": {
-    "@fewlines/eslint-config": "3.1.2",
-    "@types/jest": "28.1.1",
-    "@types/node": "17.0.40",
-    "@types/pg": "8.6.5",
-    "@types/uuid": "8.3.4",
-    "@typescript-eslint/eslint-plugin": "5.27.0",
-    "@typescript-eslint/parser": "5.27.0",
-    "eslint": "8.17.0",
-    "eslint-config-prettier": "8.5.0",
-    "eslint-plugin-import": "2.26.0",
-    "eslint-plugin-prettier": "4.0.0",
-    "jest": "28.1.0",
-    "prettier": "2.6.2",
-    "ts-jest": "28.0.4",
-    "ts-node": "10.8.1",
-    "typescript": "4.7.3",
+    "@fwl/database": "file:../database",
+    "uuid": "8.3.2",
```

Comment:
Why are we keeping dependencies in some `packages/<package>/package.json`, since we install them from the workspace, then?

Reply:
for publishing, we don't want `uuid` in the dependencies of `logging`

Reply:
So [my first thought](https://github.com/fewlinesco/node-web-libraries/pull/358#discussion_r892082108) was right?

Reply:
Hum yes, I forgot this thing about the unique `yarn.lock` for the workspace. 
It's like installing globally (but for the workspace only) every package. 
And, it always uses the last version of our packages to (we did not mention that, but it's cool)

Reply:
yes ☝️